src/semant/translate.sml

```diff
@@ -150,4 +151,7 @@ structure Translate : TRANSLATE = struct
                                  T.LABEL(join)]),
                                T.CONST(0))))
     end
+
+  fun sequence(l: exp list) =
+    Ex(Tree.ESEQ(l))
```

Comment:
@soelgary oh boy... I thought I pushed, but it failed because there were upstream changes. We've been talking about old code. I'm sorry, I'll push in one sec as soon as I rebase.